drivers/usb/device/usb_dc_stm32.c

```diff
@@ -469,13 +469,17 @@ int usb_dc_attach(void)
 	 * DM00310109.
 	 */
 #ifdef PWR_CR2_USV
+#if defined(LL_APB1_GRP1_PERIPH_PWR)
```

Comment:
This isn't defined for STM32WBX, but the reference manually still indicates that V(ddusb) needs to be enabled on startup, so I added the `else` here. Not sure if this is 100% correct or not, but is working for me here.

Reply:
This is fine.